src/main/java/framework/components/Header.java

```diff
@@ -0,0 +1,42 @@
+package framework.components;
+
+import framework.helpers.WebDriverWaitHelper;
+import framework.pages.AbstractPage;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class Header {
+    @FindBy(xpath = "//a[@class='ico-login']")
+    WebElement loginIcon;
+
+    @FindBy(xpath = "//a[@class='ico-account']")
+    WebElement accountIcon;
+
+    @FindBy(xpath = "//h1")
+    WebElement caption;
+
+    WebDriverWaitHelper waitHelper;
+    public Header(WebDriver driver) {
+        waitHelper = new WebDriverWaitHelper(driver);
+        PageFactory.initElements(driver, this);
+    }
+
+    @Step("Verify that user is not logged in")
+    public boolean isUserNotLogin(){
+        return loginIcon.isDisplayed();
+    }
```

Comment:
As for me, there is no need to create this method as you can use isUserLoggedIn() instead.

Reply:
Removed this method.